charts/events/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: events
 description: Observe kubernetes event collection
 type: application
-version: 0.1.18
```

Comment:
Would it be more accurate to bump minor version here given you removed a field? What will happen if folks follow current examples which do reference useJob?